packages/odyssey/src/scss/components/_tag.scss

```diff
@@ -13,15 +13,16 @@
 // limitations under the License.
 
 .ods-tag {
-  display: inline-block;
   margin: 0 $spacing-xs $spacing-xs 0;
   padding: 0 $spacing-xs-em;
   border-radius: $base-border-radius;
-  background: cv('gray', '200');
+  background: cv('gray', '100');
```

Comment:
Is it essential that this PR contains these changes? We should prioritize keeping them outside of this PR unless absolutely necessary.